MovieQuiz/Models/GameRecord.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  GameRecord.swift
+//  MovieQuiz
+//
+//  Created by Богдан Полыгалов on 14.11.2022.
+//
+
+import Foundation
+
+struct GameRecord: Codable, Comparable {
+    let correct: Int //количество правильных ответов
+    let total: Int //количество вопросов квиза
+    let date: String //дата завершения раунда
+    
+    static func < (lhs: GameRecord, rhs: GameRecord) -> Bool {
+        if lhs.correct < rhs.correct{
+            return true
+        }
+        else {
+            return false
+        }
```

Comment:
Можно результат выражения использовать в качестве возвращаемого значения
```suggestion
        return lhs.correct < rhs.correct
```
А когда возвращаемое значение вычисляется за одну строчку, можно еще и return опустить:
```suggestion
       lhs.correct < rhs.correct
```